terracumber_config/tf_files/SUSEManager-4.2-NUE.tf

```diff
@@ -114,58 +114,78 @@ module "cucumber_testsuite" {
       }
     }
     server = {
-      image = "sles12sp4o"
       provider_settings = {
         mac = "aa:b2:93:01:00:81"
       }
     }
     proxy = {
-      image = "sles12sp4o"
       provider_settings = {
         mac = "aa:b2:93:01:00:82"
       }
     }
     suse-client = {
-      name = "cli-sles12"
+      image = "sles15sp1o"
+      name = "cli-sles15"
       provider_settings = {
-        mac = "aa:b2:93:01:00:83"
+        mac = "aa:b2:93:01:00:84"
```

Comment:
Shouldn't this be this? : 
```
suma-42-cli-sles15        { hardware ethernet aa:b2:92:03:00:c4; fixed-address 10.84.220.196; option host-name "suma-42-cli-sles15";        }
```

Reply:
I think you mixed NUE and PRV?

NUE:
https://gitlab.suse.de/galaxy/infrastructure/-/blob/master/srv/salt/dhcpd-server/mgr_suse_de/dhcpd.conf#L158
```
host suma-42-cli-sles15             { hardware ethernet aa:b2:93:01:00:84; fixed-address 10.162.210.132; option host-name "suma-42-cli-sles15";            }
``